less/styles.less

```diff
@@ -14,12 +14,77 @@
 @import '~eclipsefdn-solstice-assets/less/quicksilver/openmdm/styles.less';
 @import '_variables.less';
 
+.custom-jumbotron {
+  background-image: url('/images/openmdm-slideshow-bg.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: top;
+  min-height: 350px;
+  border-top: 1px solid #ccc;
 
-.header-wrapper .featured-jumbotron{
-  background: @jumbotron-bg-color;
-  padding-top:48px;
-  p:last-child{
-    margin-bottom:0;
+  h2 {
+    font-weight: 700;
+    font-size: 2.7em;
+  }
+
+  p {
+    font-size: 1.25em;
+  }
+}
+
+.vertical-center {
```

Comment:
We should already have this class definition in the base theme, so you should be able to remove this one

Reply:
this one is only for vertical center, not horizontal center. I have checked "vcenter" class, not quite like this I want since it's using inline-block

Reply:
But I will see since I probably gonna change this jumbotron to use something like IOT

Reply:
https://github.com/EclipseFdn/solstice-assets/blob/master/less/_components/classes-grid.less#L36

Reply:
Yeah, noticed this. What I was doing since I did not want horizontal center, but keep vertical center. But now I have dropped this since this is not the best way to do so, m referring to IOT's better method .